// @vidstack/react/dist/dev/player/vidstack-default-layout.js
"use client"

export { defaultLayoutIcons } from './vidstack-default-icons.js';
export { k as DefaultAirPlayButton, c as DefaultAudioLayout, h as DefaultBufferingIndicator, o as DefaultCaptionButton, w as DefaultChapterTitle, C as DefaultChaptersMenu, z as DefaultControlsSpacer, q as DefaultFullscreenButton, l as DefaultGoogleCastButton, a as DefaultLayoutContext, x as DefaultLiveButton, n as DefaultMuteButton, p as DefaultPIPButton, m as DefaultPlayButton, r as DefaultSeekButton, E as DefaultSettingsMenu, y as DefaultTimeGroup, A as DefaultTimeInfo, B as DefaultTimeInvert, t as DefaultTimeSlider, v as DefaultTitle, j as DefaultTooltip, g as DefaultVideoGestures, D as DefaultVideoKeyboardActionDisplay, e as DefaultVideoLargeLayout, d as DefaultVideoLayout, f as DefaultVideoSmallLayout, s as DefaultVolumeSlider, i as i18n, u as useDefaultLayoutContext, b as useDefaultLayoutWord } from '../chunks/vidstack-BLjNh3_d.js';
import 'react';
import '../chunks/vidstack-DpFFzXBw.js';
import '../chunks/vidstack-Xvq9wjeH.js';
import '../chunks/vidstack-DXuea2hX.js';
import '../chunks/vidstack-BqbbXnmy.js';
import '../chunks/vidstack-BBPG_wWY.js';
import '../chunks/vidstack-BhjICVce.js';
import 'react-dom';
import '../chunks/vidstack-BHpALDxJ.js';
import '../chunks/vidstack-DwYRrObt.js';
