// @vidstack/react/dist/dev/player/vidstack-default-components.js
"use client"

export { k as DefaultAirPlayButton, c as DefaultAudioLayout, h as DefaultBufferingIndicator, o as DefaultCaptionButton, w as DefaultChapterTitle, C as DefaultChaptersMenu, z as DefaultControlsSpacer, q as DefaultFullscreenButton, l as DefaultGoogleCastButton, x as DefaultLiveButton, n as DefaultMuteButton, p as DefaultPIPButton, m as DefaultPlayButton, r as DefaultSeekButton, E as DefaultSettingsMenu, y as DefaultTimeGroup, A as DefaultTimeInfo, B as DefaultTimeInvert, t as DefaultTimeSlider, v as DefaultTitle, j as DefaultTooltip, g as DefaultVideoGestures, e as DefaultVideoLargeLayout, d as DefaultVideoLayout, f as DefaultVideoSmallLayout, s as DefaultVolumeSlider } from '../chunks/vidstack-BLjNh3_d.js';
import 'react';
import '../chunks/vidstack-DXuea2hX.js';
import '../chunks/vidstack-BqbbXnmy.js';
import '../chunks/vidstack-BBPG_wWY.js';
import '../chunks/vidstack-BhjICVce.js';
import 'react-dom';
import '../chunks/vidstack-BHpALDxJ.js';
import '../chunks/vidstack-DwYRrObt.js';
