// @vidstack/react/dist/prod/player/vidstack-default-components.js
"use client"

export { k as DefaultAirPlayButton, c as DefaultAudioLayout, h as DefaultBufferingIndicator, o as DefaultCaptionButton, w as DefaultChapterTitle, C as DefaultChaptersMenu, z as DefaultControlsSpacer, q as DefaultFullscreenButton, l as DefaultGoogleCastButton, x as DefaultLiveButton, n as DefaultMuteButton, p as DefaultPIPButton, m as DefaultPlayButton, r as DefaultSeekButton, E as DefaultSettingsMenu, y as DefaultTimeGroup, A as DefaultTimeInfo, B as DefaultTimeInvert, t as DefaultTimeSlider, v as DefaultTitle, j as DefaultTooltip, g as DefaultVideoGestures, e as DefaultVideoLargeLayout, d as DefaultVideoLayout, f as DefaultVideoSmallLayout, s as DefaultVolumeSlider } from '../chunks/vidstack-CYAsr6Gb.js';
import 'react';
import '../chunks/vidstack-CvduS_bi.js';
import '../chunks/vidstack-Caa7In1T.js';
import '../chunks/vidstack-DD0YAtlF.js';
import '../chunks/vidstack-BthB6HyT.js';
import 'react-dom';
import '../chunks/vidstack--kGe51hP.js';
import '../chunks/vidstack-Nxbu5UbW.js';
