// @vidstack/react/dist/prod/player/vidstack-default-layout.js
"use client"

export { defaultLayoutIcons } from './vidstack-default-icons.js';
export { k as DefaultAirPlayButton, c as DefaultAudioLayout, h as DefaultBufferingIndicator, o as DefaultCaptionButton, w as DefaultChapterTitle, C as DefaultChaptersMenu, z as DefaultControlsSpacer, q as DefaultFullscreenButton, l as DefaultGoogleCastButton, a as DefaultLayoutContext, x as DefaultLiveButton, n as DefaultMuteButton, p as DefaultPIPButton, m as DefaultPlayButton, r as DefaultSeekButton, E as DefaultSettingsMenu, y as DefaultTimeGroup, A as DefaultTimeInfo, B as DefaultTimeInvert, t as DefaultTimeSlider, v as DefaultTitle, j as DefaultTooltip, g as DefaultVideoGestures, D as DefaultVideoKeyboardActionDisplay, e as DefaultVideoLargeLayout, d as DefaultVideoLayout, f as DefaultVideoSmallLayout, s as DefaultVolumeSlider, i as i18n, u as useDefaultLayoutContext, b as useDefaultLayoutWord } from '../chunks/vidstack-CYAsr6Gb.js';
import 'react';
import '../chunks/vidstack-DpFFzXBw.js';
import '../chunks/vidstack-Xvq9wjeH.js';
import '../chunks/vidstack-CvduS_bi.js';
import '../chunks/vidstack-Caa7In1T.js';
import '../chunks/vidstack-DD0YAtlF.js';
import '../chunks/vidstack-BthB6HyT.js';
import 'react-dom';
import '../chunks/vidstack--kGe51hP.js';
import '../chunks/vidstack-Nxbu5UbW.js';
